import-export-cli/cmd/keys.go

```diff
@@ -81,12 +83,19 @@ func getKeys() {
 	tiers, err := getAvailableAPITiers(accessToken)
 
 	if tiers != nil && err == nil {
-		utils.Logln(utils.LogPrefixInfo + "Retrieved available subscription tiers of the API: ", tiers)
-		//Needs an available subscription tier when creating application
-		throttlingTier = tiers[0]
+		utils.Logln(utils.LogPrefixInfo+"Retrieved available subscription tiers of the API: ", tiers)
```

Comment:
Shall we add a space between LogPrefixInfo + "Retrieved available subscription tiers of the API: "?